src/routes/docs/products/functions/quick-start/+page.markdoc

```diff
@@ -46,12 +46,26 @@ The easiest way to execute your first function is to use the Appwrite Console.
 # Explore {% #explore %}
 Use this your first function as a springboard to explore the flexible and powerful features of Appwrite Functions.
 
-| Explore                                                                 |                                                    |
-|-------------------------------------------------------------------------|----------------------------------------------------|
-| [Develop](/docs/products/functions/development)                | Learn about how to parse request, build responses, and log to Console in Appwrite Functions. |
-| [Response](/docs/products/functions/development#response)       | Learn about function response types, like text, json, html, redirect, and empty responses.|
-| [Deploy](/docs/products/functions/deployment)                | Learn to deploy Appwrite Functions from Git, the CLI, or manual uploads. |
-| [Execute](/docs/products/functions/execution)            | Appwrite Functions can be executed in a variety of ways, like SDK, domains, schedules, events, and more. Learn about executions.  |
-| [Domains](/docs/products/functions/execution#domains)       | Each Appwrite Function can be executed through it's own domain, generated or custom. Learn about domains.  |
-| [Runtime](/docs/products/functions/runtimes)            | Appwrite supports over a dozen runtimes to meet your preferences. Learn about runtimes.  |
-| [Examples](/docs/products/functions/examples)            | Enjoy learning from examples? Take a look at these examples functions.  |
+{% cards %}
+{% cards_item href="/docs/products/functions/development" title="Develop" %}
+Learn how to parse request, build responses, and log to Console in Appwrite Functions.
+{% /cards_item %}
+{% cards_item href="/docs/products/functions/development#response" title="Response" %}
+Learn about function response types, like text, json, html, redirect, and empty responses.
+{% /cards_item %}
+{% cards_item href="/docs/products/functions/deployment" title="Deploy" %}
+Learn to deploy Appwrite Functions from Git, the CLI, or manual uploads.
+{% /cards_item %}
+{% cards_item href="/docs/products/functions/execution" title="Execute" %}
+Appwrite Functions can be executed in a variety of ways, like SDK, domains, schedules, events, and more. Learn about executions.
+{% /cards_item %}
+{% cards_item href="/docs/products/functions/execution#domains" title="Domains" %}
```

Comment:
![CleanShot 2024-02-02 at 13 45 50@2x](https://github.com/appwrite/website/assets/19310830/29510b40-a5b5-4876-8144-1ed0c25d62a6)

I tried and most of the links don't work

Reply:
Oh yeah I broke all these.